pkg/kubelet/container/runtime.go

```diff
@@ -349,6 +353,8 @@ type Image struct {
 	RepoDigests []string
 	// The size of the image in bytes.
 	Size int64
+	// ImageSpec for the image which include annotations.
+	Spec ImageSpec
```

Comment:
This seems like another case where we could remove the `ID` field as it is duplicated in `Spec`.

Reply:
Discussed here: https://github.com/kubernetes/kubernetes/pull/90061#discussion_r416217160